components/integrations/installed_incoming_webhooks/installed_incoming_webhooks.tsx

```diff
@@ -161,6 +173,9 @@ export default class InstalledIncomingWebhooks extends React.PureComponent<Props
                 }
                 searchPlaceholder={Utils.localizeMessage('installed_incoming_webhooks.search', 'Search Incoming Webhooks')}
                 loading={this.state.loading}
+                nextPage={this.nextPage}
+                previousPage={this.previousPage}
```

Comment:
no need for specific handler functions

```suggestion
                nextPage={() => this.setState({page: page + 1})}
                previousPage={() => this.setState({page: page - 1})}
```